last_seen.py

```diff
@@ -11,23 +11,39 @@ def seen_evt(self, event):
         events = ['quit', 'part', 'kick', 'join', 'rename', 'topic', 'message']
         if event.event_type in events:
             nick = event.meta['nick']
-            self.seen[nick] = (event.event_type, gmtime())
+            self.seen[nick] = (event.event_type, datetime.datetime.utcnow())
 
     @cmd
     def last_seen(self, event):
-        """
-        Returns when was given user last seen.
+        """Return when was given user last seen.
 
         Example: !last_seen <user>
         """
         args = event.args
         if len(args) < 1:
-            return "Name argument missing."
+            return 'Name argument missing.'
         user = args[0]
 
         if user not in self.seen:
-            return "This user was never here."
+            return 'This user has never been here.'
```

Comment:
@Adman this implies that we only check whether the user has been in the channel/room this is running in. That is certainly not the case.

Would it be better to say for instance `'We haven't seen {0} yet.'.format(user)`?